src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
```

Comment:
Pretty easy to mock the AI and gameData with mockito.
```
@Mock
GameData mockGameData
@Mock
ProAI mockAI

when( mockGameData.getGameData()).thenReturn(mockAI)
when(mockAI...()).thenReturn()
```

Since the interactions are limited to just the `ai`, the mocking is pretty straight forward. The single AI object is not a bad thing IMO, it hides the data and provides some utility around it. Plus it actually can make for decent mocking since you can use some pretty decent high level functionality with it, better than just having primitives to work with.